app/models/state_file_base_intake.rb

```diff
@@ -407,12 +412,23 @@ def latest_submission
   end
 
   def increment_failed_attempts
+    # If unlock_in time has passed, unlock the intake. Otherwise, the intakes' failed_attempts will
+    # remain unchanged until the verification code is answered correctly (see: unlock_for_login!)
+    # derived from https://github.com/heartcombo/devise/issues/4679#issuecomment-1453450974
+    if lock_expired?
+      unlock_access!
+    end
```

Comment:
do we actually need this? by the time they get here, their lock should have been cleared after putting in their verification code